src/marqo/s2_inference/clip_utils.py

```diff
@@ -231,4 +232,38 @@ def encode(self, inputs: Union[str, ImageType, List[Union[str, ImageType]]],
         else:
             logger.debug('text')
             return self.encode_text(inputs, normalize=normalize)
+
+class OPEN_CLIP(CLIP):
+    def __init__(self, model_type: str = "open_clip/ViT-B-32-quickgelu/laion400m_e32", device: str = 'cpu',  embedding_dim: int = None,
+                            truncate: bool = True, **kwargs) -> None:
+        super().__init__(model_type, device,  embedding_dim, truncate , **kwargs)
+        self.model_type = model_type.split("/", 3)[1]
```

Comment:
maybe use `model_name` and keep `model_type` as the full string. i guess `model_type` is already being stored from the parent class

Reply:
`self.model_name = model_type.split("/", 3)[1]`